dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/python/PythonGateway.java

```diff
@@ -575,6 +575,14 @@ public Resource queryResourcesFileInfo(String userName, String fullName) {
         return resourceService.queryResourcesFileInfo(userName, fullName);
     }
 
+    /**
```

Comment:
Actually we could remove this comment, since the logic is quite clear and comment with external link is hard to maintain.

Reply:
Yeah, I think we should remove issue link too, but I personally do not mind add comment or not